src/composables/useTour.ts

```diff
@@ -0,0 +1,393 @@
+                },
+            },
+            ui: {
+                elementsWithOpacity: [
+                    '.address-overview .mobile-action-bar',
+                ],
+                elementsWithoutInteractivity: [
+                    '.address-overview .actions-mobile',
+                    '.address-overview .active-address',
+                ],
+                disabledNextStep: true,
+            },
+        },
+        [MobileOnboardingTourStep.FIRST_TRANSACTION]: {
+            path: '/transactions',
+            tooltip: {
+                target: '.transaction-list .list-element > .transaction > .identicon',
+                content: [
+                    "Here's your first transaction with your first NIM.",
+                    'Every NIM address comes with an avatar. They help to make sure you got the right one.',
+                ],
+                params: {
+                    placement: 'bottom-start',
+                },
+            },
+            lifecycle: {
+                onMountedStep: () => {
+                    const buyNimBtn = document
+                        .querySelector('.address-overview .transaction-list a button') as HTMLButtonElement;
+                    if (!buyNimBtn) return;
+                    buyNimBtn.disabled = true;
+                },
+            },
+            ui: {
+                elementsWithOpacity: [
+                    '.address-overview .mobile-action-bar',
+                ],
+                elementsWithoutInteractivity: [
+                    '.address-overview',
+                ],
+            },
+        },
+        [MobileOnboardingTourStep.BITCOIN_ADDRESS]: {
+            path: '/',
+            tooltip: {
+                target: '.account-overview .bitcoin-account > .bitcoin-account-item > svg',
+                content: [
+                    'This is your Bitcoin wallet. You get one with every Nimiq account.',
+                ],
+                params: {
+                    placement: 'top-start',
+                },
+            },
+            ui: {
+                elementsWithOpacity: [
+                    '.account-overview .backup-warning',
+                    '.account-overview .mobile-menu-bar',
+                    '.account-overview .account-balance-container',
+                    '.account-overview .address-list',
+                    '.account-overview .mobile-action-bar',
+                ],
+                elementsWithoutInteractivity: [
+                    '.account-overview .bitcoin-account',
+                ],
+            },
+        },
+        [MobileOnboardingTourStep.WALLET_BALANCE]: {
+            path: '/',
+            tooltip: {
+                target: '.account-overview .account-balance-container .amount',
+                content: [
+                    'Check the bar-chart to see how your addresses compose your total balance.',
+                    'Currently you have 100% NIM, and no BTC.',
```

Comment:
This percentage should probably be dynamic to the actual value shown.

Reply:
Totally